### ISSUE TITLE
Learn how to import CNN pretrained Caffe models to DIGITS 

### ISSUE THREAD

**Issue by elboyran** [nlesc-sherlock/deeplearning#9]
See the [NVIDIA issue](https://github.com/NVIDIA/DIGITS/issues/49)

**Comment by Maartenvm** [comment]
This issue is co-assigned to @elboyran

**Comment by mongoose54** [comment]
Is there some tutorial on how to load caffe models (e.g [Model zoo](https://github.com/BVLC/caffe/wiki/Model-Zoo)) on DIGITS? It seems that Thanks!

**Comment by egpbos** [comment]
I'm getting the same error as the last commenter ("ERROR: must specify a loss layer")

**Comment by egpbos** [comment]
I tried with the lotsacars dataset combined with the lotsacars-20151202-170935-03d3 model and editing the pasted prototxt text like it says here:
https://github.com/NVIDIA/DIGITS/issues/360#issuecomment-147778970
Something is running now, but accuracy is at ~70%, which seems too low. It's possible that I'm not matching up the right dataset with the right model, as there are multiple lotsacars datasets and models...

**Comment by egpbos** [comment]
Found the right combination:
- lotsacars 300 dataset with ID 20151201-173123-e279
- lotsacars max300 sigmoid 0.005 with ID 20151202-170935-03d3

Made successful clone run! 90% accuracy. ID 20160510-133810-a71f on my DAS DIGITS instance.

Steps taken:
- Create new Model in DIGITS (New Model -> Classification)
- Select right dataset (lotsacars in my case)
- Options:
  - Training epochs: 1
  - Base Learning Rate: 0
  - under "Show advanced learning rate options" set Policy: Fixed
- Pick a nice model name at the bottom of the page.
- The important part: find the `train_val.prototxt` from your model and copy the text into the Custom Network field.
- The _more_ important part: search for `data_param` in the pasted text from the previous step. Comment out the `data_param.source` and `data_param.backend` lines (see https://github.com/NVIDIA/DIGITS/issues/360#issuecomment-147778970).
- The _even more_ important part: find the path of the snapshot file (in the filesystem where DIGITS is running) of the trained model (usually something like `snapshot_iter_######.caffemodel`) and put it in the Pretrained model field.

**Comment by egpbos** [comment]
Note: this was for a pre-trained model **from** DIGITS, back into DIGITS.
